Makefile

```diff
@@ -111,8 +111,17 @@ staticcheck:
 test:
 	$(GOTEST) -v ./...
 
+.PHONY: test-coverprofile
+test-coverprofile:
+	go test -covermode=count -coverprofile=coverage.out `go list ./... | grep -v integration` -json > test-report.out
+
+.PHONY: coverage
+coverage:
+	go test -covermode=count -coverprofile=coverage.out `go list ./... | grep -v integration`
+	go tool cover -html=coverage.out
+
 .PHOHY: checks
```

Comment:
Noticed a typo here from the original makefile H/N 

Would it be cleaner to have a single PHONY directive at the top of the file ?
Reduces repetition

Reply:
Yeah, maybe. I don't really mind. 👍